spec/requests/api/v1.0/portfolios_spec.rb

```diff
@@ -435,6 +435,14 @@
         end
       end
 
+      context 'bad portfolio' do
+        let(:params) { {:name => tag_name} }
+        it 'returns 404' do
+          post "#{api_version}/portfolios/1515151515/tag", :headers => default_headers, :params => tag_params
```

Comment:
I think we can use `#{portfolio.id +1}` instead of `1515151515`, in the unlikely event that the portfolio gets generated with that id, haha. Then if we ever create another portfolio in this spec, it would fail, but I think that's fine because we're then ensuring that we would need to put in an ID that doesn't exist.